deeprank_gnn/models/conservation.py

```diff
@@ -22,8 +20,11 @@ def get_conservation(self, amino_acid):
 class ConservationTable:
     "holds data for one position-specific scoring table"
 
-    def __init__(self, rows={}):
-        self._rows = rows
+    def __init__(self, rows=None):

```

Comment:
It's dangerous to use an empty dict as the default value. If you modify a default argument, it will persist until the next call, so your "empty" dict will start to contain values on calls other than the first one.

Reply:
You're right!